loader_hub/file/audio/base.py

```diff
@@ -17,23 +17,40 @@ class AudioTranscriber(BaseReader):
 
     """
 
-    def __init__(self, *args: Any, model_version: str = "base", **kwargs: Any) -> None:
+    def __init__(self, *args: Any, model_version: str = "base", 
+                model_type: str = "openai_whisper", diarization_max_speakers: str = None,
+                language: str = None, language_behaviour: str = "automatic multiple languages",
+                target_translation_language: str = 'english', gladia_api_key: str = None, 
```

Comment:
To make this interface a bit cleaner, could we create a separate dataclass as an AudioConfig? there could be two subclasses, with openai and gladia as the two:

```python
@dataclass
class WhisperAudioConfig(BaseAudioConfig):
    """Whisper audio config."""
    model_version: str


@dataclass
class GladiaAudioConfig(BaseAudioConfig):
    """Gladia Audio Config.""" 
    gladia_api_key: str
    diarization_max_speakers: str
    language: str
    language_behaviour: str
    ...
```

Then have the `__init__` of the AudioTranscriber take in the model_type and an audio_config arg (we could keep model_version too to keep it backwarsd compatible)